articles/explainable-ai/index.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/explainable-ai/
+title: Explainable AI - An Introduction
+description: This article will go over
+author: collins-ayuya
+date: 2020-10-20T00:00:00-11:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/explainable-ai/hero.jpg
+    alt: Explainable AI example image
+---
+AI-powered systems have a lot of influence on our daily lives. A number of these systems are so sophisticated that little to no human intervention is required in their design and deployment. These systems make a lot of decisions for us every single day. They make a lot of crucial decisions that affect human lives (for example, in medicine, finance, law, defense). But how they do is often not explained.
+<!--more-->
+
+### Introduction
+There is a growing need to understand how these systems make decisions. We will be exploring the need for an explainable AI.
+
+#### Table of Contents
+1. Defining explainable AI.
+
+2. Need for explainable AI.
+
+3. A few use cases of explainable AI.
+
+4. XAI techniques.
+
+5. Challenges to achieving explainable AI.
+
+#### Prerequisites
+To get the most from this article a basic understanding of artificial intelligence concepts will be useful. To understand some basic machine learning concepts check out this [article](/engineering-education/supervised-learning-algorithms/). A few concepts of machine learning and deep learning are also covered [here](/engineering-education/automated-fake-news-detection/).
+
+#### A few useful terms
+**Black box model** – a data mining and machine learning model whose internals are not interpretable to humans. The model internals are unknown to the observer. It could also be a proprietary model.
+
+**Neural network** – a series of algorithms that are modeled on the human brain, used to identify underlying relationships in data.
```

Comment:
Rewrite as: 'a series of algorithms modeled on the human brain used to identify underlying data relationships.' for conciseness.